pyomo/dataportal/TableData.py

```diff
@@ -176,35 +180,34 @@ def _set_data(self, headers, rows):
 
         elif self.options.format == 'set_array':
             if not self.options.index is None:
-                msg = "Cannot specify index for data with the 'set_array' "   \
-                      'format: %s'
+                msg = "Cannot specify index for data with the 'set_array' " 'format: %s'
```

Comment:
String formatting (looks like it was a mix of single and double quotes to begin with which is why your filter didn't pick it up)